### DISCUSSION TITLE
stump grinding function?

### DISCUSSION THREAD

**Discussion by puppryde** [Courseplay/Courseplay_FS22#3150] [Ideas]
i enjoy running tree orchards on my farm, but find stump grinding to be extremely tedious, so i'm curious if it would be possible to have a stump grinding function, in which the route would seek out and drive directly into tree stumps within the field bounds. for ease of use, i'd imagine this would be simplest to do with relatively wide, front mounted tools such as on the primoth raptor. i'd imagine the function would be pretty much the same as the bale seeking ai.

**Comment by Tensuko** [comment]
Not possible and from my on experience, stumps dont always disappear when move to them or over them.
Too unreliable.
